### PR TITLE
feat(icons): add lectern

### LINKED ISSUE #2021: Icon request: speaking podium / lectern
### Icon name

lectern

### Use cases

For use with UI related to formal public speaking, e.g. debates, galas, commemorative events etc.
For use with UI related to shows, e.g. comedy, standup, etc.
For use as representation of an immobile alternative to the very generic `mic-vocal` icon.

### Design ideas

I made a draft in Figma which roughly adheres to the design guidelines, but I'm not sure it's at the appropriate level of quality.
image

### Checklist

- [X] I have searched if someone has submitted a similar issue before and there weren't any. (Please make sure to also search closed issues, as this issue might already have been resolved.)
- [X] I have searched existing icons to make sure it does not already exist and I didn't find any.
- [X] I am not requesting a brand logo and the art is not protected by copyright.
- [X] I am not requesting an icon that includes religious, political imagery or hate symbols.
- [X] I have provided appropriate use cases for the icon(s) requested.

### PR DESCRIPTION
**Pull request by gurtt**
Closes #2021 

## What is the purpose of this pull request?

- [x] New Icon
- [ ] Bug fix
- [ ] New Feature
- [ ] Documentation update
- [ ] Other:

### Description

Adds `lectern` icon.

### Icon use case 

From #2021 :
* For use with UI related to formal public speaking, e.g. debates, galas, commemorative events etc.
* For use with UI related to shows, e.g. comedy, standup, etc.
* For use as representation of an immobile alternative to the very generic `mic-vocal` icon.

### Alternative icon designs 

This design specifically depicts a lectern. Designs with a square base or platform should be separate pulpit and podium icons, respectively. Tags added since these terms are often confused.

### Concept 

- [x] I have provided valid use cases for each icon.
- [x] I have not added any a brand or logo icon.
- [x] I have not used any hate symbols.
- [x] I have not included any religious or political imagery.

### Author, credits & license

- [x] The icons are solely my own creation.
- [ ] The icons were originally created in #<issueNumber> by @<githubUser>
- [ ] I've based them on the following Lucide icons: 
- [ ] I've based them on the following design: 

### Naming 

- [x] I've read and followed the [naming conventions](https://lucide.dev/guide/design/icon-design-guide#naming-conventions)
- [x] I've named icons by what they are rather than their use case.
- [x] I've provided meta JSON files in `icons/[iconName].json`.

### Design 

- [x] I've read and followed the [icon design guidelines](https://lucide.dev/guide/design/icon-design-guide)
- [x] I've made sure that the icons look sharp on low DPI displays.
- [x] I've made sure that the icons look consistent with the icon set in size, optical volume and density.
- [x] I've made sure that the icons are visually centered.
- [x] I've correctly optimized all icons to three points of precision.

## Before Submitting 

- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

**Comment by karsa-mistmere** [comment]
> This design specifically depicts a lectern. Designs with a square base or platform should be separate pulpit and podium icons, respectively. Tags added since these terms are often confused.

I'm very much afraid you've have fallen victim to the same confusion (either that, or it's an Aussie thing). 😅

Lecterns were at some point started to be mistakenly called podiums (the confusion probably rising from a lectern also being placed on top of the podium for the speaker to read notes from) and by now this usage has been naturalised (much to my woe, but it's still an irrefutably prevalent usage, especially in the US). Pretty much the same thing happened to the word pulpit (although to a much lesser extent).

Lecterns themselves can be of pretty much any shape or size (short, tall, thin, wide), although the one you designed is most reminiscent of what is sometimes called a speaker stand (which is confusing in its own right, since the term "speaker stand" also refers to what electric speakers are placed on).

One thing is for sure: we won't be adding separate variants with various base widths since it's extremely unlikely for these variants to have legitimate use cases of their own, and should we add a `podium` icon, it will depict a sports podium, which again is a completely different kettle of fish.

This being said, I think it would be nice to explore some alternate designs, especially since I feel like lecterns with wide bases are much more the norm than flimsy speaker stands are. 🙂

File: icons/lectern.svg
Comment (karsa-mistmere) [inline icons/lectern.svg:1]: The icon is very noticeably off centre vertically, to the point that I have to wonder if maybe it was by accident and one of these were the original intent? 🤔
image

Comment (gurtt) [reply icons/lectern.svg:1]: Between this and the origami issue I think there's something up with my tooling. Original intent (and how it appears in my editor) is the top-right variant (although the bottom-right looks better).

**Comment by gurtt** [comment]
I suspect it may be an Aussie thing 😅 I asked a colleague about this and they confirmed what I had said so I suspect it varies by dialect.

In my original comment I should've said that the difference (as I/we understand it) between a lectern and a pulpit is that, essentially, you can hide behind a pulpit. They're larger and surround you to some extent usually, whereas a lectern is just a stand.

Either way, some alternate designs with heftier stands:
lectern-a
lectern-b
lectern-c

The second one is an example that's more front-on rather than slightly from-below, but it looks like a drink cup :/

**Comment by karsa-mistmere** [comment]
> I suspect it may be an Aussie thing 😅 I asked a colleague about this and they confirmed what I had said so I suspect it varies by dialect.
> 
> In my original comment I should've said that the difference (as I/we understand it) between a lectern and a pulpit is that, essentially, you can hide behind a pulpit. They're larger and surround you to some extent usually, whereas a lectern is just a stand.
> 
> Either way, some alternate designs with heftier stands: lectern-a lectern-b lectern-c
> 
> The second one is an example that's more front-on rather than slightly from-below, but it looks like a drink cup :/

I vote for the last one, I find that it's probably the most legible out of the two wide ones, to me that one immediately screams `lectern`.

**Comment by ericfennis** [comment]
I agree with @karsa-mistmere, the last one works well. 
@gurtt Can you post a 100% scale version?

**Comment by jguddas** [comment]
Could be sharper.

**Comment by jguddas** [comment]
> > Could be sharper.
> 
> What do you mean?
> 
> Edit: just to clarify, we'll be going with this design image

Can you push this, I'm not sure if it has the same sharpness problem?

**Comment by karsa-mistmere** [comment]
> > > Could be sharper.
> > 
> > 
> > What do you mean?
> > Edit: just to clarify, we'll be going with this design image
> 
> Can you push this, I'm not sure if it has the same sharpness problem?

It wasn't my design but I've painstakingly recreated it in a mega-optimized version:

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M16+12h3a2+2+0+0+0+1.902-1.38l1.056-3.333A1+1+0+0+0+21+6H3a1+1+0+0+0-.958+1.287l1.056+3.334A2+2+0+0+0+5+12h3%22+/%3E%3Cpath+d=%22M18+6V3a1+1+0+0+0-1-1h-3%22+/%3E%3Crect+x=%228%22+y=%2210%22+width=%228%22+height=%2212%22+rx=%221%22+/%3E&name=image">icons<br/>Open lucide studio</a>

**Pull request closed without merging**